src/config/dynamic-theme-fixes.config

```
@@ -13726,7 +13726,8 @@ img[src*="/images/images2/icons/"]
 t.bilibili.com
 
 CSS
-#app {
+#app, .fixed-bg, .fixed-bg[data-v] {
```

```suggestion
#app,
.fixed-bg {
```
`.fixed-bg[data-v]` is unnecessary